requirements.txt

```diff
@@ -1,2 +1,3 @@
 ansible
 netaddr
+configparser>=3.3.0
```

Comment:
in the case of this inventor_builder only, shouldn't we localize this dep within the script ?